src/components/PlacesAutocomplete.tsx

```diff
@@ -21,24 +21,24 @@ const PlacesAutocomplete: React.FC = () => {
   })
 
   function getSuggestionsElements() {
-    const listElements = document.getElementsByTagName('li')
+    const listElements = document.getElementsByTagName('button')
     if (listElements.length === 0) {
       return null
     }
     return Array.from(listElements)
   }
 
-  function clearLastSuggestion() {
+  function clearLastSuggestion(): void {
```

Comment:
No need to type function return values unless needed. Leverage TypeScript's type inference.

```suggestion
  function clearLastSuggestion() {
```